packages/pipeline-editor/src/PipelineEditorWidget.tsx

```diff
@@ -62,14 +62,17 @@ import { IntlProvider } from 'react-intl';
 
 import { PIPELINE_CURRENT_VERSION } from './constants';
 import * as i18nData from './en.json';
+import { formDialogWidget } from './formDialogWidget';
 import * as palette from './palette.json';
-import { PipelineExportDialog } from './PipelineExportDialog';
+
+import PipelineExportDialog from './PipelineExportDialog';
```

Comment:
imports should be wrapped in `{ }`, I'm not sure why this is passing lint, in fact this change chuck is unnecessary in general.